### PR TITLE
Support dynamic height container

### PR DESCRIPTION
**Pull request by zlangbert**
For #6. Let me know what you think

### CODE REVIEW

File: docs/src/Main.jsx [hunk 1]
```diff
@@ -138,6 +139,23 @@ const Main = React.createClass({
             </div>
           </SwipeableViews>
 
+          <h3>Demo 4</h3>
+          <p>
```
Comment (oliviertassinari) [inline docs/src/Main.jsx]: :+1: good example

**Comment by oliviertassinari** [comment]
I'm gonna have the same feedback for the coding style. I will add an excellent task.
Regarding the result, I can't test it until tomorrow.
I really appreciate the time you spent on this :+1:

**Comment by zlangbert** [comment]
Fixed all eslint errors.

Np! I was about to create the same type of thing myself, glad I stumbled on this

**Comment by oliviertassinari** [comment]
Thanks for this :+1:

**Pull request merged**